checkdmarc/help.md

```diff
@@ -214,11 +223,23 @@ This action will check DMARC records against alternate name servers.
 
 ##### Input
 
-|Name|Type|Default|Required|Description|Enum|
-|----|----|-------|--------|-----------|----|
-|domain|string|None|True|Domain to check. e.g. fbi.gov, google.com in alternate nameserver|None|
-|nameservers|[]string|['1.1.1.1', '1.0.0.1']|True|Nameserver to check against. e.g ["1.1.1.1","1.0.0.1"]|None|
-|timeout|number|6.0|True|Timeout in seconds for request. Default is 6 seconds|None|
+|Name|Type|Default|Required|Description|Enum|Example|
+|----|----|-------|--------|-----------|----|-------|
+|domain|string|None|True|Domain to check. e.g. fbi.gov, google.com in alternate nameserver|None|rapid7.com|
+|nameservers|[]string|['1.1.1.1', '1.0.0.1']|True|Nameserver to check against. e.g ["1.1.1.1","1.0.0.1"]|None|["1.1.1.1","1.0.0.1"]|
+|timeout|number|6|True|Timeout in seconds for request. Default is 6 seconds|None|6|
+
+Example input:
+
+```
+{
+  "domain": "whitehouse.gov",
```

Comment:
@mhofert-r7 Can we get this to match the example key? Future versions of icon-plugin will just generate from that key.